src/Storage/Media/MediaStorage.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Drupal\discoverable_entity_bundle_classes\Storage\Media;
+
+use Drupal\discoverable_entity_bundle_classes\Storage\SqlContentEntityStorageTrait;
+use Drupal\media\MediaStorage as MediastorageBase;
```

Comment:
@jaybabak , Can you up the `as` to match case of the class used below? Should be `MediaStorageBase`

Reply:
Yes dang - don't know how I missed that!